src/containers/App.jsx

```diff
@@ -15,6 +15,7 @@ import { NavBar } from "../components/common/NavBar";
 function App() {
   return (
     <React.Fragment>
+      learn react
```

Comment:
I believe there is no need for `learn react` coz we have `yarn run test --passWithNoTests` in the config file